app/components/dwd-rainradar.js

```diff
@@ -0,0 +1,44 @@
+//=========== This component provided DWD Rain Radar Overlay
+import Component from '@ember/component';
+import { alias } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
+
+import TileLayer from 'ol/layer/Tile';
+import TileWMS from 'ol/source/TileWMS';
+
+export default class extends Component {
+  @service aircraft;
+  @service ddb;
+  @service filter;
```

Comment:
those services don't seem to be used anywhere in the component

Reply:
Removed unneeded services